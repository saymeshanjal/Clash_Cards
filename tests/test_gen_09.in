17
14 29 2
16 4 3
2 31 1
14 23 1
8 46 3
12 30 3
4 39 5
2 15 3
14 9 1
16 14 2
10 9 3
16 14 2
8 9 3
14 1 3
16 2 5
10 15 2
16 4 1