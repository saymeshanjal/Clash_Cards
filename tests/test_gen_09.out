75 318 3039
495 12375 0
1 1 0
1594 1838 0
27 43 0
180 744 0
0 6 0
0 2 0
1558 1874 0
593 2530 9747
17 235 0
593 2530 9747
4 66 0
0 3432 0
0 12870 0
7 41 204
4719 8151 0
